frontend/src/widgets/Overview.js

```diff
@@ -83,14 +83,14 @@ Overview.propTypes = {
 };
 
 Overview.defaultProps = {
-  data: PropTypes.shape({
```

Comment:
Curious about this change - didn't realize PropTypes.shape was incorrect

Reply:
I did too when I first wrote it! :upside_down_face: